src/util/commandline/app-command.ts

```diff
@@ -45,40 +45,8 @@ export class AppCommand extends Command {
 
     return result;
   }
-}
-
-export function getCurrentApp(optValue: string): ResultOrValue<DefaultApp> {
```

Comment:
I remember needing this function for something somewhere, but don't remember where. Is it really unused now?